doc/source/user_guide/io.rst

```diff
@@ -3947,6 +3952,22 @@ in files and will return floats instead.
    Currently pandas only supports *reading* binary Excel files. Writing
    is not implemented.
 
+.. _io.calamine:
+
+Calamine (Excel and ODS files)
+------------------------------
+
+The :func:`~pandas.read_excel` method can read Excel file (``.xlsx``, ``.xlsm``, ``.xls``, ``.xlsb``)
+and OpenDocument spreadsheets (``.ods``) using the ``python-calamine`` module.
+This module is a binding for Rust library `calamine <https://crates.io/crates/calamine>`__
+and faster then other engines in most cases. The semantics and features for reading files
+match what can be done for `Excel files`_ using ``engine='calamine'``.
```

Comment:
I don't understand what the sentence "The semantics and features..." is trying to convey.

Reply:
Removed it.